bundles/core/org.eclipse.smarthome.core.thing.xml.test/src/test/resources/test-bundle-pool/ChannelTypesTest.bundle/ESH-INF/thing/thing-types.xml

```diff
@@ -26,6 +26,7 @@
 	
 	<channel-group-type id="channelgroup">
 		<label>Channel group</label>
+		<category>TEMPERATURE</category>
```

Comment:
Jus curious what is the preferred case here the PR contains both pure uppercase TEMPERATURE and more human readable Temperature